### DISCUSSION TITLE
<Bad file list formatting>

### DISCUSSION THREAD

**Discussion by sovahc** [community/community#68097] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi! Why you show me blank fields instead of full file names?
Ipad mini 5, Latest IOS, Safari.
![IMG_4209](https://github.com/community/community/assets/7010920/3ab69e4b-daf7-4684-b69b-e162e449b183)

**Comment by tiagoangelos** [comment]
This happens due to the size of the screen, so the responsiveness 'hides' these details depending on the size of your screen! i hope it help!
